R/features.R

```diff
@@ -16,22 +16,33 @@
 #' @keywords internal
 #'
 #' @author Nikolai Sellereite, Martin Jullum
-feature_combinations <- function(m, exact = TRUE, noSamp = 200, shapley_weight_inf_replacement = 10^6, reduce_dim = TRUE) {
+feature_combinations <- function(m, exact = TRUE, n_combinations = 200, shapley_weight_inf_replacement = 10^6, reduce_dim = TRUE) {
+
+  # Force user to use a natural number for n_combinations if m > 10
```

Comment:
Suggest using m > 12 instead. This is 8000 and should be perfectly fine. 

Suggesting to adjust the text to 

Due to the computational complexity, we recommend setting n_combinations to 10 000 instead of the exact method when the number of features is larger than 12. To force use the exact method, set n_combinations equal to 2^m, where m is the number of features.